samples/nrf9160/nrf_cloud_mqtt_multi_service/Kconfig

```diff
@@ -112,14 +112,22 @@ config LOCATION_TRACKING_GNSS
 	default y
 	depends on LOCATION_METHOD_GNSS
 	help
-	  Disable both this and LOCATION_TRACKING_CELLULAR to completely disable location tracking.
+	  Disable all location tracking methods to completely disable location tracking.
 
 config LOCATION_TRACKING_CELLULAR
 	bool "Use cellular method for location tracking"
 	default y
 	depends on LOCATION_METHOD_CELLULAR
 	help
-	  Disable both this and LOCATION_TRACKING_GNSS to completely disable location tracking.
+	  Disable all location tracking methods to completely disable location tracking.
+
+config LOCATION_TRACKING_WIFI
+	bool "Use Wi-Fi method for location tracking"
+	default n
+	depends on LOCATION_METHOD_WIFI
+	help
+	  Requires the use of an nRF7002 companion chip.
```

Comment:
should this explicitly state:
`nRF9160 DK with an attached nRF7002 EK Wi-Fi evaluation kit` ?

Reply:
I don't think so, since technically it can work on any hardware that has `nRF7002` at its disposal and properly installed in the device tree